nengo_loihi/builder.py

```diff
@@ -812,7 +814,10 @@ def signal_probe(model, key, probe):
     if kwargs is not None:
         if kwargs['function'] is not None:
             raise BuildError("Functions not supported for signal probe")
-        weights = kwargs['transform'].T / model.dt
+
+        assert isinstance(kwargs["transform"], Dense)
```

Comment:
I would do a `BuildError` here rather than an assert, since a user could reasonably trip this.

Reply:
I wasn't anticipating that we'd merge this in before supporting Convolutions, so these are just here as a temporary reminder that this assumes Dense transforms.

Reply:
Ok, sounds good!